src/auth/PrivateRoute.tsx

```diff
@@ -15,18 +17,34 @@ interface PrivateRouteProps {
  * Represents a protected route that can only be accessed when the user is
  * authenticated. Users will be redirected to the Auth0 login page if they
  * attempt to access protected routes.
+ *
+ * If this route should only be accessible by organizers, please set the
+ * `admin` prop to True.
  */
 const PrivateRoute: React.FC<PrivateRouteProps> = ({
   component,
   path,
+  admin,
   exact,
   sensitive,
   strict,
 }: PrivateRouteProps) => {
   const { user } = useAuth0()
-  const route: React.ComponentType<any> = user?.email_verified
-    ? component
-    : Verify
+
+  let route: React.ComponentType<any> = component
+
+  // redirect if email is not verified
+  if (!user?.email_verified) {
+    route = VerifyView
+  }
+
+  // if admin route, redirect if not admin
+  const namespace = "https://example.com"
```

Comment:
@krishnakpandian I can't seem to find where this came from. This doesn't seem to match the `audience`, and I couldn't find it on Auth0. Is this something that we can access with react-auth0?

Reply:
This is with a hook auth0 supports in terms of using a config. Right now, it seems to default to example.com and not retrieve the audience but I'll try updating to see if I can get it to support using the audience. That's just what Auth0 default hook is, we can probably update the field name to be something easier to parse like `roles` if you think that might be better as well. I'll try having it fixed by tomorrow.

Reply:
Just did some quick testing and looked through some documentation but it seems the namespace is required. Let's just use `https://cruzhacks.com` as our namespace since it doesn't actually refer to the resource it's accessing.

Reply:
Ahh, makes sense. Thanks for looking into this!